addons/mail/static/tests/discuss/core/web/command_palette.test.js

```diff
@@ -87,22 +84,19 @@ test("Channel mentions in the command palette of Discuss app with @", async () =
             [".o_command.focused .o_command_name", { text: "Mitchell Admin and Mario" }],
         ],
     });
-    await contains(".o_command_palette .o_command_category", {
-        contains: [[".o_command_name", { text: "Mitchell Admin" }]],
-    });
```

Comment:
We shouldn't lose the ability to open chat with oneself. It's useful, and especially since it was possible before, users might have personal content inside it.